Tweak.xm

```diff
@@ -214,6 +214,14 @@ static void bootstrapPiP(YTPlayerViewController *self, BOOL playPiP, BOOL killPi
 
 %end
 
+%hook YTPlayerResponse
+
+- (BOOL)isPlayableInPictureInPicture {
+    return YES;
+}
+
+%end
```

Comment:
Are you sure there is `-isPlayableInPictureInPicture` method inside `YTPlayerResponse`?
![image](https://user-images.githubusercontent.com/3608783/123531851-2e4bfa80-d732-11eb-9693-85a903e11562.png)

Reply:
The only reasonable candidate is this one but it also already gets hooked.
![image](https://user-images.githubusercontent.com/3608783/123531866-5e939900-d732-11eb-9355-844b33a19c7a.png)

Reply:
> Are you sure there is `-isPlayableInPictureInPicture` method inside `YTPlayerResponse`?
> ![image](https://user-images.githubusercontent.com/3608783/123531851-2e4bfa80-d732-11eb-9693-85a903e11562.png)

Yeah, I have YT 16.24.2 in IDA Pro and there is that method: 
![image](https://user-images.githubusercontent.com/43968044/123551723-fc8e6000-d740-11eb-97c6-d7788708ddd3.png)

Reply:
@SarahH12099 I don't see it for some reasons.. What the body of that method looks like?
![image](https://user-images.githubusercontent.com/3608783/123564271-ffda2800-d7e2-11eb-9fa3-3d1fed0e1a85.png)

Reply:
![image](https://user-images.githubusercontent.com/3608783/123570948-6bc48c80-d7f3-11eb-9be2-37654a24b4e8.png)
Hooking is not needed. It gets the value from `YTIPlayabilityStatus`.